modules/analysis/rosbag-image-pipeline/images/object-detection/requirements.txt

```diff
@@ -0,0 +1,42 @@
+boto3
```

Comment:
Did you generate these requirements using `pip-compile requirements.in`?
As a standard, we declare the requirements in `requirements.in` and use pip compile to generate `requirements.txt`?

Reply:
Hey Srini, no I got the requirements file from one of our data scientists and didn't want to touch it. Let me give that a try